public/scripts/tokenizers.js

```diff
@@ -147,6 +147,24 @@ async function resetTokenCache() {
     }
 }
 
+/**
+ * Maps tokenizer IDs to their names.
+ * @example { 0: 'none', 1: 'gpt2', ... }
+ */
+export const TOKENIZER_NAME_MAP = Object.fromEntries(
```

Comment:
Isn't `tokenizers` aready an exported enum that contains both friendly names and the id/value that is needed?
I mean doesn't hurt, so it's okay, but if it's not much more verbose to write simple `.map` inside the slash command enum and callback, we don't need to make another list here.

Generally saying, as much as I don't like going directly for UI elements in backend code.. but as we already need to set and trigger the event on the `#tokenizer` anyway, you *could* even just query the option list from that. It contains both the exact ids you need, and also the friendly names, already in a formatted way that works nice to present in the slash command.

Reason why I suggest this: The `tokenizers` map does contain more tokenizers than the ones that are available in the UI. Likely on purpose, though @Cohee1207 would have to say that.
The option 2: **OPENAI** is missing.

So if you select that one from your slash command, it'll set it to null and that leads to errors.